WalletWasabi.Gui/UiConfig.cs

```diff
@@ -98,5 +101,8 @@ public string LockScreenPinHash
 			get => _lockScreenPinHash;
 			set => RaiseAndSetIfChanged(ref _lockScreenPinHash, value);
 		}
+
+		[JsonProperty(PropertyName = "CoinListViewSortingPreference")]
+		public SortingPreference CoinListViewSortingPreference { get; internal set; } = new SortingPreference(SortOrder.Increasing, default(SortOrder));
```

Comment:
this property is being reset/overwritten to the default value at startup for some reason, any ideas @lontivero @yahiheb ?

Reply:
`JsonConverter` is missing for `SortingPreference`. Try to do something like this: https://github.com/molnard/WalletWasabi/blob/4ba03984edb774d61d68c1e0cb2b7bb957bb683a/WalletWasabi/CoinJoin/Common/Crypto/SchnorrPubKey.cs#L38

Reply:
Okay that worked! Thanks @molnard :D